docs/en/snapshotcloneserver_en.md

```diff
@@ -0,0 +1,165 @@
+
+2. Create temporary snapshot on CurveFS, retuen seqNum of the snapshot created, then update the seqNum of the snapshot record.
+4. Create snapshot mapping table (see 1.4.1), and save it to object storage S3.
+6. Store snapshot data from CurveFS to object storage S3. When storing, the snapshot system first reads the snapshot data from CurveFS, then uploads the data to the S3.
+8. Delete the temporary snapshot.
+10. Update snapshot status to 'done'.
+
+### 1.4 Organization of Snapshot Data
+
+Each snapshot has a corresponding record in the snapshot system and a persisted one in Etcd, and a uuid as the unique identifier.
+
+#### 1.4.1 Organization of snapshot data on S3
+
+Snapshot data is stored in S3 in chunks, and each chunk corresponds to an object on S3. The object that stores data is called a data object, and each snapshot also has an meta object that records the metadata. The meta object records the file information of the snapshot and the mapping table between the snapshot chunk and the object. The meta object consists of 2 parts:
+
+1. Snapshot file information, including original data volume name, volume size and chunk size, etc.
+2. Snapshot chunk mapping table, recording the list of snapshot data objects.
+
+Figure 2 is a sample of the sanpshot data.
+
+<p align="center">
+    <img src="../images/snap-s3-format.png"><br>
+    <font size=3> Figure 3: Snapshot data format</font>
+</p>
+
+### 1.5 Interface of Curve Snapshot System
+
+Please refer to the [document](../cn/snapshotcloneserver_interface.md) of snapshot and clone.
+
+## 2、Clone
+
+### 2.1 Intro to Curve Clone Module
+
+In section 1 we introduced the snapshot system, which is for creating snapshots of  files of CurveFS and incrementally dump the data to S3. But for a complete snapshot system, supporting only the snapshot creation is not enough, file recovery and cloning is the reason why take snapshot. In this section we introduce Curve cloning (recovering can be considered as cloning to some extend).
+
+According to the data source, there are two kinds of cloning, including cloning from snapshot and cloning from mirror, and if dividing by whether the data is entirely cloned before the service is provided, it can be divided into delayed cloning and non-delayed cloning.
```

Comment:
delayed cloning and non-delayed cloning    -》  Lazy Clone and Not-Lazy Clone

Reply:
> delayed cloning and non-delayed cloning -》 Lazy Clone and Not-Lazy Clone

done